modules/ROOT/pages/dataweave-extension-plugin.adoc

```diff
@@ -0,0 +1,397 @@
+= Installing and Using the DataWeave 2.0 Extension
+
+Install and use DataWeave 2.0 extension to Visual Studio Code (DataWeave extension) to develop, preview, test, and deploy custom mappings and modules into libraries in a DataWeave project. The extension also enables you to preview transformations similar to those you perform in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground].
+
+== Before You Begin
+
+Before installing the DataWeave 2.0 extension to Visual Studio Code, you must:
+
+* Install AdoptOpenJDK (Java 8 minimum)
+* Install Apache Maven (3.6.3 minimum)
```

Comment:
I updated these based on your comments, let me know if the minimum versions are correct @martincousido.
+ @dukesphere for visibility on her doc.

Reply:
@kevintroller Looks like this PR says a requirement is "Java 8 or 11 provided," but not sure what that is supposed to mean. https://github.com/mulesoft/mule-dx-data-weave-component/pull/168/files Would a later version of Java work? And I actually installed Oracle Java 11, and it seems to work.

Reply:
Hi @martincousido  @TomoDC Could you reconfirm the correct requirements to install before installing the extension? It is:

- AdoptOpenJDK (which minimum version) or JDK (Java 8 or 11)

Thank you